.github/workflows/run-end-to-end.yml

```diff
@@ -230,7 +235,9 @@ jobs:
         DD_API_KEY: ${{ secrets.DD_API_KEY }}
     - name: Run APPSEC_MISSING_RULES scenario
       if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"APPSEC_MISSING_RULES"')
-      run: ./run.sh APPSEC_MISSING_RULES
+      run: |
+        # nginx does not start with missing rules file, thereby stalling the job
```

Comment:
could we add this condition to "if:"?
`      if: always() && steps.build.outcome == 'success' && contains(inputs.scenarios, '"APPSEC_MISSING_RULES"') && matrix.weblog != 'nginx'
`

Reply:
I can try